src/WebUI/Pods/PodsTable.tsx

```diff
@@ -51,62 +52,66 @@ export class PodsTable extends BaseComponent<IPodsTableProperties> {
                     headingContent={
                         <div>
                             <h3 className={"heading-title"}>{this.props.headingText}</h3>
-                            <span className={"secondary-text"}>{this._generateHeadingSubText(this.statusCount)}</span>
+                            <span className={"secondary-text pods-table-subtext"}>{this._generateHeadingSubText(this.statusCount)}</span>
                         </div>
                     }
-                    className={css("list-content", "pl-details", "depth-16")}
+                    className={css("list-content", "pl-details", "depth-16", "pods-table")}
                     items={this.props.podsToRender}
-                    columns={PodsTable._getColumns(this.props.showWorkloads || false)}
+                    columns={PodsTable._getColumns(this.props.showWorkloadsColumn || false)}
                     onItemActivated={this._showPodDetails}
+                    showLines={true}
                 />
             );
         }
 
         return null;
     }
 
-    private static _getColumns(showWorkloads: boolean): ITableColumn<V1Pod>[] {
+    private static _getColumns(showWorkloadsColumn: boolean): ITableColumn<V1Pod>[] {
```

Comment:
i am assuming you have taken care of the widths based on the figma, if not we should follow

Reply:
yes

Reply:
yes